boot/bootutil/src/caps.c

```diff
@@ -47,3 +47,12 @@ uint32_t bootutil_get_caps(void)
 
         return res;
 }
+
+uint32_t bootutil_get_num_images(void)
+{
+#if defined(MCUBOOT_IMAGE_NUMBER)
+	return MCUBOOT_IMAGE_NUMBER;
```

Comment:
tab -> spaces

Reply:
Given that the file already has a mix of tabs and spaces, maybe I should just have a separate patch after this that fixes it all to spaces.

Reply:
Or just change it all together now

Reply:
I guess the other approach is to start with a patch that fixes the existing tabs, and then make this patch correct.